internal/jennies/all.go

```diff
@@ -50,8 +52,10 @@ func (languageJennies LanguageJennies) AsLanguageRefs() []string {
 func All() LanguageJennies {
 	return LanguageJennies{
 		golang.LanguageRef:     golang.New(),
+		java.LanguageRef:       java.New(),
```

Comment:
are we adding Java now?

Reply:
It was already in the list, the items just got reordered a bit :)

Btw: this list is the list of supported languages, it doesn't mean Java will be published to the foundation sdk straight away